lib/js/components/SelectList/SelectList.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+	<div class="selectList">
+		<div
+			v-for="(item, index) in items"
+			:key="index"
```

Comment:
i know that we have many indexes as key, but i'm not a fan of this solution, look to https://vueschool.io/articles/vuejs-tutorials/tips-and-gotchas-for-using-key-with-v-for-in-vue-js-3/ at "No indexes as keys"